packages/tools/devtools/devtools-browser-extension/e2e-tests/tsconfig.cjs.json

```
@@ -0,0 +1,12 @@
+{
+	// This config must be used in a "type": "commonjs" environment. (Use fluid-tsc commonjs.)
+	"extends": "./tsconfig.json",
+	"compilerOptions": {
+		"outDir": "../../dist/e2e-tests",
```

Same as previous comment
```suggestion
		"outDir": "../dist/e2e-tests",
```